app/components/ErrorBoundary/ErrorBoundary.css

```diff
@@ -1,22 +1,4 @@
 .container {
-  display: flex;
-  justify-content: center;
-}
-
-.snap {
-  border: 1px var(--danger-color) dashed;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  margin: 10px;
-  padding: 10px;
-
-  & img {
-    height: 100px;
-    width: 100px;
-  }
-}
-
-.message {
-  margin-left: 10px;
+  width: 40%;
```

Comment:
Have you tested how smol this can get on phones? Obviously I haven't either - but I feel like this could benefit from a media query: width: 90% or something like that

Reply:
Yes you're right - didn't think of that hehe